app/src/main/java/fr/free/nrw/commons/utils/ViewUtil.java

```diff
@@ -32,6 +38,36 @@ public static void showShortSnackbar(View view, int messageResourceId) {
             }
         });
     }
+    public static void showLongSnackbar(View view, String text) {
```

Comment:
Is this function used in this PR?

Reply:
Oh!! my bad,
I added the implementation in another branch before. 
Just committed the changes to this one :)